x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/test/SecuritySingleNodeTestCase.java

```diff
@@ -109,35 +104,6 @@ protected void before() {
         }
     };
 
-    /**
-     * A JUnit class level rule that runs after the AfterClass method in {@link ESIntegTestCase},
-     * which stops the cluster. After the cluster is stopped, there are a few netty threads that
-     * can linger, so we wait for them to finish otherwise these lingering threads can intermittently
-     * trigger the thread leak detector
-     */
-    @ClassRule
-    public static final ExternalResource STOP_NETTY_RESOURCE = new ExternalResource() {
```

Comment:
This is an `ESSingleNodeTestCase` which doesn't derive from `ESIntegTestCase` so I think we are missing another call to `awaitGlobalNettyThreadsFinish` here.

Reply:
Fair point, I'll just add it to `ESSingleNodeTestCase` also, so we cover all of those as well since they may spin up Netty tasks here and there when using REST.

Reply:
++ even if not using REST we still bind to a transport port in these tests.